service/controller/v2/resource/chart/desired.go

```diff
@@ -2,12 +2,20 @@ package chart
 
 import (
 	"context"
+	"encoding/json"
 
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/giantswarm/chart-operator/service/controller/v2/key"
 )
 
+const (
+	ValuesData = "values.json"
```

Comment:
Pawel's opinions:

1. It shouldn't be exported. It makes is possible to use it outside the package, therefore brings confusion. Also slows down tooling and compilation time.
2. When it's used in a single place it even shouldn't be a const. It only makes readers jump up and down w/o actual benefit of reusability.

Reply:
I rarely agree with Pawel's opinions but I like them anyway. Just wanted to say that. :D <3

Reply:
Good point. I get a bit carried away with the constants sometimes. I've removed since its only used once.